Description: Add more info to script man page
 Most of this path was taken from 
 lib/HTML/FormHandler/Generator/DBIC.pm
Author: dod
--- a/script/form_generator.pl
+++ b/script/form_generator.pl
@@ -19,7 +19,30 @@
 
 =head1 NAME
 
-dbic_form_generator - form generator
+dbic_form_generator - Generate HTML form from DB schema with DBIC
+
+=head1 DESCRIPTION
+
+Using the database passed in the command option, dbic_form_generator
+will query the DB for a schema and generate a HTML form to update the
+content of the DB.
+
+Options:
+
+  rs_name       -- Resultset Name
+  schema_name   -- Schema Name
+  db_dsn        -- dsn connect info
+  class_prefix  -- [OPTIONAL] Prefix for generated classes (Default: '')
+  label         -- [OPTIONAL] Flag to toggle generation of form labels (Default: 0)
+  label_column  -- [OPTIONAL] Flag to toggle generation of dummy form labels_columns for type: 'select' (Default: 0)
+
+This package should be considered still experimental since the output,
+of the generated classes will be changed from time to time.  This should
+not impact the main usage for this module that we had in mind, that is
+generating the initial version of a FormHandler form class, copying
+it to the project and modifying it.
+
+This script is installed into the system with the rest of FormHandler.
 
 =head1 VERSION
 
